galera/Project1/galera.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+#include<string>
+
+using namespace std;
+
+class galera
+{
+private:
+	string name;
```

Comment:
Please, make some syntax distinction for data members. e.g. you could prefix them with `m_` or with `_`